environ.py

```diff
@@ -1,25 +1,28 @@
 # standard libraries
-import json, logging, os
+import json
+import logging
+import os
 from json.decoder import JSONDecodeError
+from typing import Union
 
+# Set up ENV
 # entry-level job modules need to be one-level beneath root
-ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+ROOT_DIR: str = os.path.dirname(os.path.abspath(__file__))
+CONFIG_DIR: str = os.path.join(ROOT_DIR, os.getenv('ENV_DIR', os.path.join('config', 'secrets')))
+CONFIG_PATH: str = os.path.join(CONFIG_DIR, os.getenv('ENV_FILE', 'env.json'))
 
 logger = logging.getLogger(__name__)
 
-
-# Set up ENV
-
-config_path = os.path.join(ROOT_DIR, os.getenv('ENV_PATH', os.path.join('config', 'secrets', 'env.json')))
-
 try:
-    with open(config_path) as env_file:
-        ENV = json.loads(env_file.read())
+    with open(CONFIG_PATH) as env_file:
+        ENV: dict = json.loads(env_file.read())
 except FileNotFoundError:
-    logger.error('Configuration file could not be found; please add env.json to the config directory.')
+    logger.error(
+        'Configuration file could not be found; please add env.json to the config directory.')
     ENV = None
 
-logging.basicConfig(level=ENV['LOG_LEVEL'])
+LOG_LEVEL: Union[str, int] = ENV.get('LOG_LEVEL', 0)
```

Comment:
I'd prefer (to keep the typing simpler) if the default was `"INFO"` or `"DEBUG"`.

Reply:
I'll switch it to `'INFO'`.

Reply:
✅Fixed.